components/testimony/TestimonyDetailPage/TestimonyDetailPage.tsx

```diff
@@ -28,10 +38,42 @@ export const TestimonyDetailPage: FC = () => {
           </Col>
 
           <Col md={4}>
-            {!isMobile && <PolicyActions className="mb-4" isUser={isUser} />}
+            {!isMobile && (
+              <PolicyActions
+                className="mb-4"
+                isUser={isUser}
+                isReporting={isReporting}
+                setReporting={handleReporting}
+              />
+            )}
             <RevisionHistory />
           </Col>
+          {/* Report Modal */}
+          {isReporting && (
+            <ReportModal
+              onClose={() => setIsReporting(false)}
+              onReport={report => {
+                reportMutation.mutate({
+                  report,
+                  testimony: revision
+                })
+              }}
+              isLoading={reportMutation.isLoading}
+              additionalInformationLabel="Additional information:"
+              reasons={[
+                "Personal Information",
+                "Offensive",
+                "Violent",
+                "Spam",
+                "Phishing"
+              ]}
+            />
+          )}
+          {/*  */}
```

Comment:
Missing comment?

Reply:
I'm sorry, could you elaborate?

Reply:
Just on line 72, there's just an empty comment, so I didn't know if you wanted to write something there, or maybe I'm just missing something.

Reply:
Oh! yes. My apologies. I had used that to point out the end of {/*Report Modal*/}
I should have removed it or made it more clear

Reply:
No worries! Just wanted to make sure.

Reply:
Understood, I appreciate it